app/routes/docs.sample._index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+handle:
+  breadcrumb: Sample Code
+---
+
+# Sample code for GCN Notices
+
+This sample code demonstrates how to handle GCN Notices in both JSON and VOEvent XML formats, and some samples from the FAQs section of the [gcn-kafka-python](https://github.com/nasa-gcn/gcn-kafka-python) repository.
+
+## Apache Kafka and GCN Streaming
+
+[Apache Kafka](https://kafka.apache.org) is an open-source distributed event streaming platform used by GCN to stream Notices and Circulars in real time.
+
+## GCN Notices Formats and Distributions
+
+[GCN Notices](https://gcn.nasa.gov/docs/notices) provide real-time machine readable alerts and are distributed in different formats:
+
+- New GCN Notices are provided in JSON format and are streamed via Kafka. Note that SVOM notices are only available via VOEvent over Kafka and email.
+- GCN Classic Notices are provided in text, binary and VOEvent XML, and are distributed both via Kafka and email.
+
```

Comment:
Remove headers for "Apache Kafka ..." and "GCN Notices Formats ...". Then move "This sample code ..." to after them, so that this is all background info for the sample code.